Mathlib/MeasureTheory/Measure/Lebesgue/Complex.lean

```diff
@@ -31,11 +32,26 @@ def measurableEquivPi : ℂ ≃ᵐ (Fin 2 → ℝ) :=
   basisOneI.equivFun.toContinuousLinearEquiv.toHomeomorph.toMeasurableEquiv
 #align complex.measurable_equiv_pi Complex.measurableEquivPi
 
+@[simp]
+theorem measurableEquivPi_apply (a : ℂ) :
+    measurableEquivPi a = Fin.cons a.re (Fin.cons a.im finZeroElim) := rfl
```

Comment:
Is `![]` notation not available here?

Reply:
Indeed, it works here and makes things easier to read. Thanks